### PR TITLE
DBZ-639 Removing restriction for none snapshot locking mode so that it can be…

### PR DESCRIPTION
**Pull request by vamossagar12**
… used for all snapshot modes

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java [hunk 1]
```diff
@@ -1073,7 +1071,7 @@ private static int validateSnapshotLockingMode(Configuration config, Field field
             return 1;
         }
 
-        // Determine the snapshot mode defined.
+        /*// Determine the snapshot mode defined.
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: Why is this just commenting out this code? If we want to go that route, the check should be removed completely.

**Comment by vamossagar12** [comment]
@gunnarmorling yeah I had kept it commented thinking would remove it once we are ok with the changes. Have removed all the commented code now.

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @vamossagar12. Could you also send a PR for updating the [connector docs](https://github.com/debezium/debezium.github.io/blob/develop/docs/connectors/mysql.asciidoc) accordingly?

**Comment by vamossagar12** [comment]
@gunnarmorling Thanks. Sure would do that. There's another issue I had opened regarding the behaviour with MyISAM engine tables. Here it is:
https://issues.jboss.org/browse/DBZ-652?filter=-2

Should I mention that as well in the docs as part of the PR sent to the documentation?

**Pull request closed without merging**